styles.css

```diff
@@ -0,0 +1,61 @@
+body {
+  background-color: violet;
+}
+
+#container {
+  display: flex;
+  flex-direction: column;
+  padding: 10rem;
+}
+
+.game-title {
+  align-self: center;
+  color: rgb(63, 63, 63);
+  font-family: Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 2em;
+  text-align: justify;
+}
+
+.cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  perspective: 1000px;
+  align-self: center;
+}
+
+.card {
+  width: 300px;
+  height: 400px;
+  position: relative;
+  cursor: pointer;
+  transform-style: preserve-3d;
+  transition: transform 0.6s;
+}
+
+.flipped {
+  transform: rotateY(180deg);
+}
+
+.card img {
+  width: 100%;
+  height: 100%;
```

Comment:
Let's keep an eye on these styles in future weeks. In general you want to be careful about setting `img` elements to specific heights/widths as it can cause them to distort if their dimensions don't match the DOM element perfectly.

For now I think it's fine, but once we start adding more images, we might need to revise these styles.

Reply:
I updated that part, hope it is better now.
! But I already completed the next task for week 2. I have no problems with the grid of cards or the images.